pkg/features/kube_features.go

```diff
@@ -834,6 +834,13 @@ const (
 	//
 	// Enable MinDomains in Pod Topology Spread.
 	MinDomainsInPodTopologySpread featuregate.Feature = "MinDomainsInPodTopologySpread"
+
+	// owner: @aravindhp @LorbusChris
+	// kep: http://kep.k8s.io/2271
+	// alpha: v1.24
+	//
+	// Enables viewing logs of node services using kubectl log
+	NodeLogs featuregate.Feature = "NodeLogs"
```

Comment:
```suggestion
	NodeLogViewer featuregate.Feature = "NodeLogViewer"
```

Also suggest using a more descriptive name here, once the feature gate is added we won't be able to change it

Reply:
done